### PR TITLE
Fix the issue when gemm inputs are from subview ops

### PR DESCRIPTION
**Pull request by yzhang93**

### CODE REVIEW

File: compiler/plugins/target/AMD-AIE/Translation/AIESerializer.cpp [hunk 1]
```diff
@@ -354,6 +354,14 @@ FailureOr<std::string> getOperandStr(Value operand,
     return failure();
   }
 
+  while (!scope.symbolTable.count(operand)) {
```
Comment (MaheshRavishankar) [inline compiler/plugins/target/AMD-AIE/Translation/AIESerializer.cpp:357]: This looks similar to the `walkSubViews`. Can we just call that method?

File: compiler/plugins/target/AMD-AIE/Translation/AIESerializer.cpp [hunk 2]
```diff
@@ -627,7 +635,7 @@ LogicalResult AccelSerializer::processOperation(scf::ForOp forOp,
   }
 
   for (Operation &op : *forOp.getBody()) {
-    if (failed(processOperation(&op, subScope))) {
+    if (failed(processOperation(&op, scope))) {
```
Comment (MaheshRavishankar) [inline compiler/plugins/target/AMD-AIE/Translation/AIESerializer.cpp:638]: This change shouldnt be needed? subScope should have everything scope has + induction variable.

**Review by MaheshRavishankar — CHANGES_REQUESTED** [review]

File: compiler/plugins/target/AMD-AIE/Translation/AIESerializer.cpp (hunk 2 above)
Comment (yzhang93) [reply compiler/plugins/target/AMD-AIE/Translation/AIESerializer.cpp:637]: I have to use `scope` to make indentation correct for the contents within the for loop.

Comment (MaheshRavishankar) [reply compiler/plugins/target/AMD-AIE/Translation/AIESerializer.cpp:637]: But scope wont have the induction variable... surprising that the indentation goes off if you use subscope... 
If its just an indentation issue, could you change this to subScope again. Maybe something else is off with the indentation, but thats cosmetic.

**Review by MaheshRavishankar — APPROVED** [review]
Approved based on using subscope again.

**Pull request merged**